pyproject.toml

```diff
@@ -162,6 +162,8 @@ select = [
 ignore = [
   # Allow non-abstract empty methods in abstract base classes
   "B027",
+  # Allow use of datetime with tz and date.today
+  "DTZ005", "DTZ011",
```

Comment:
I think we don't need `DTZ005`. The only remaining use is for the `period` in the ZIM filename which can be replaced with a `date.today`